internal/indexer/definitions/subsplease.yaml

```diff
@@ -46,11 +46,13 @@ irc:
     type: single
     lines:
       - test: 
-          - "[Release] [SubsPlease] Title - 01 (480p) [636F636B].mkv (420.69MB) - https://nyaa.si/view/0000000 - https://nyaa.si/view/0000000/torrent"
-        pattern: '\[Release\] (.*(SubsPlease).*?)\.?(mkv)? \((\d+.?\d*[KMGTP]?B)\) - (.*) - (.*)'
+          - "[Release] [SubsPlease] Multiple Word Title - 01 (480p) [636F636B].mkv (420.69MB) - https://nyaa.si/view/0000000 - https://nyaa.si/view/0000000/torrent"
+          - "[Release] [SubsPlease] Multiple Word Title (01-12) (1080p) [Batch] (4.95GB) - https://nyaa.si/view/0000000 - https://nyaa.si/view/0000000/torrent"
+        pattern: '\[Release\] (.*(SubsPlease)] ([^-()]+) .*?\.?)(mkv)? \((\d+.?\d*[KMGTP]?B)\) - (.*) - (.*)'
```

Comment:
Creating a character class which prohibits matching of hyphens or dashes breaks matching of all titles with said characters.

Reply:
Used pattern matching instead of character class in the latest commit